sqldelight-gradle-plugin/src/test/fixtures/javadoc/src/main/sqldelight/com/sample/Test.sq

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE test (
+  _id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT
+);
+
+/**
+ Javadoc comment yo.
```

Comment:
it can be done. Is there a way to add javadoc in javapoet without it automatically putting the comments around it